### DISCUSSION TITLE
ONVIF not working on SmartPSS

### DISCUSSION THREAD

**Discussion by TheAlchemist92** [roleoroleo/yi-hack-Allwinner-v2#612] [General]
Hi
I Wanted to add my Yi camera on SmartPSS (I have a camera recorder linked to my camera security system) But when I try to add the camera with IP and port (I tried with the default port 80 and with port 3777) the response is: "can not find network host"
Any idea?

**Comment by roleoroleo** [comment]
Port is TCP ~~1000~~ 80.

**Comment by TheAlchemist92** [comment]
And why in the settings the port used is 80?

Il mer 5 apr 2023, 17:21 roleo ***@***.***> ha scritto:

**Reply by roleoroleo** [reply]
Sorry, TCP port is 80.
Wrong model...

Please check if the cam is properly configured using a standard client software like Onvif Device Manager.

**Comment by TheAlchemist92** [comment]
Onvif device menager find the camera.
I tried to add the camera directly to my NVR and it finds the camera (green spot) but if I try to see the images it doesnt work... any setting to try to use?

Il gio 6 apr 2023, 09:33 roleo ***@***.***> ha scritto:

**Reply by roleoroleo** [reply]
No settings. ONVIF should magically works.
But it's a complicated standard and there are a lot of compatibility problem.
If you want to get to the bottom of the problem I need a wireshark capture between the NVR and the cam.
I could try to analyze it.

**Comment by TheAlchemist92** [comment]
How can i do this? Can we PM in some way?

Il ven 7 apr 2023, 09:04 roleo ***@***.***> ha scritto:

**Comment by roleoroleo** [comment]
- Download tcpdump from here:
https://github.com/roleoroleo/yi-hack-utils/tree/main/Allwinner-v2

- Copy it to the cam and run it with this command:
tcpdump -i eth0 -w /tmp/sd/onvif.pcap

- Run the wizard on your NVR until you receive the error
- Stop tcpdump (CTRL-C)
- Share here the file /tmp/sd/onvif.pcap
